contracts/interchain-token-service/src/contract.rs

```diff
@@ -239,17 +252,80 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
         Ok(token_id)
     }
 
+    /// Deploys an interchain token to a remote chain.
+    ///
+    /// This function initiates the deployment of an interchain token to a specified
+    /// destination chain. It validates the token metadata, emits a deployment event,
+    /// and triggers the necessary cross-chain call.
+    ///
+    /// # Parameters
```

Comment:
Is this the convention used by rustdoc to document params? If not, look into the rust convention

Reply:
yeah, I looked up rustdoc conventions, it seems using `///` is a convention.
- https://doc.rust-lang.org/rust-by-example/meta/doc.html
- https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html
```
There are two doc comments: /// and //!.

/// documents the item following it whereas //! documents the parent item.
```